### ISSUE TITLE
Not able to open admin page

### ISSUE THREAD

**Issue by wangxf123456** [GoogleCloudPlatform/django-cloud-deploy#338]
```pytb
Error during template rendering
In template /usr/local/google/home/wangxf/Desktop/django-cloud-deploy/venv/lib/python3.6/site-packages/django/contrib/admin/templates/admin/base.html, error at line 8

Reverse for 'cloud-logs' not found. 'cloud-logs' is not a valid view function or pattern name.
```

**Comment by enriquejosepadilla** [comment]
I havent been able to reproduce this. Not sure how this happened.

**Comment by enriquejosepadilla** [comment]
@wangxf123456 thoughts?

**Comment by wangxf123456** [comment]
I have a Django application which has this problem. You can visit https://django-527123.appspot.com/admin. I also added you as the project owner. If you want I can email the source code to you,

**Comment by wangxf123456** [comment]
https://django-589916.appspot.com/en/admin/ is another example

**Comment by wangxf123456** [comment]
This only happens when deploying existing apps. Some existing apps create their own admin overwrite. So it will conflict with our changes. I am wondering are we able to make this a separate app which does not inherit admin app?

**Comment by enriquejosepadilla** [comment]
this was fixed with #376
